public/team.html

```diff
@@ -0,0 +1,636 @@
+                                    fill="white"
+                                />
+                            </svg>
+                        </a>
+                    </div>
+                </div>
+            </div>
+        </footer>
+
+        <script>
+            // Burger menus
+            document.addEventListener('DOMContentLoaded', function () {
+                // open
+                const burger = document.querySelectorAll('.navbar-burger')
+                const menu = document.querySelectorAll('.navbar-menu')
+
+                if (burger.length && menu.length) {
+                    for (var i = 0; i < burger.length; i++) {
+                        burger[i].addEventListener('click', function () {
+                            for (var j = 0; j < menu.length; j++) {
+                                menu[j].classList.toggle('hidden')
+                            }
+                        })
+                    }
+                }
+
+                // close
+                const close = document.querySelectorAll('.navbar-close')
+                const backdrop = document.querySelectorAll('.navbar-backdrop')
+
+                if (close.length) {
+                    for (var i = 0; i < close.length; i++) {
+                        close[i].addEventListener('click', function () {
+                            for (var j = 0; j < menu.length; j++) {
+                                menu[j].classList.toggle('hidden')
+                            }
+                        })
+                    }
+                }
+
+                if (backdrop.length) {
+                    for (var i = 0; i < backdrop.length; i++) {
+                        backdrop[i].addEventListener('click', function () {
+                            for (var j = 0; j < menu.length; j++) {
+                                menu[j].classList.toggle('hidden')
+                            }
+                        })
+                    }
+                }
+            })
+
+            function showMeetups() {
```

Comment:
remove? but probably from the other files too, so ideally using a separate PR @igortrsk